tests/e2e/util.go

```diff
@@ -2126,3 +2126,24 @@ func getPersistentVolumeClaimSpecForFileShare(namespace string, labels map[strin
 	pvc.Spec.AccessModes = []v1.PersistentVolumeAccessMode{accessMode}
 	return pvc
 }
+
+//deleteFcdWithRetriesForSpecificErr method to retry fcd deletion when a specific error is encountered
+func deleteFcdWithRetriesForSpecificErr(ctx context.Context, fcdID string, dsRef types.ManagedObjectReference, errToIgnore string) error {
+	var err error
+	for i := 0; i < 3; i++ {
```

Comment:
Any reason for three tries only? Why dont you use `Poll` instead to retry for a specific amount of time, like in other test cleanup functions.

Reply:
Changed to use `Poll`